### DISCUSSION TITLE
Unable to run command line tools

### DISCUSSION THREAD

**Discussion by russell-sealand** [nschloe/meshio#1024] [Q&A]
The readme says: "You can then use the command-line tools" and lists some command line tools e.g. meshio-convert, but this doesn't work for me, and I can't see any files with the command line names in the repository.  Is this a work in progress, or what am I missing? Thank you.

**Comment by nschloe** [comment]
You'll need to install the Python package, it'll install the tools for you.

**Reply by russell-sealand** [reply]
My apologies - when I re-opened PowerShell it now works, I'm not sure what was wrong. Thank you.
